### ISSUE TITLE
Axes label position changes in updateSVG

### ISSUE THREAD

**Issue by DSpale** [Don-Chris/Plot2LaTeX#7]
Hi,
first of all: Thank you for this tool! It was already a great help with some of my plots.
However, I noticed a bug concerning the axes placement in a 3D plot: When exporting my figure to SVG, the position of the axes seems to reset, which leads to the result seen in the attached zip-file (Test.fig is the original figure, as generated in MATLAB).

Via debugging, I was able to conclude that the issue seems to occur in the updateSVG local function, but my knowledge of the .svg file format is fairly limited and thus I have not been able to pinpoint the exact cause and fix it.

Is there any advice on what could help fix this issue? I also tried playing with some of the options, but to no avail.

Best regards,
Dominik
[EM_wave_vis.zip](https://github.com/user-attachments/files/16107983/EM_wave_vis.zip)

**Comment by Don-Chris** [comment]
Thanks for opening this issue, I will look into it in the next days.

**Comment by Don-Chris** [comment]
@DSpale Could you send the original figure where the axis labels are at the correct position, so that I can try to replicate the label movement? And the selected options in Plot2LaTeX are of interest too (Fontsize, etc)

I tried to reproduce your figure without any issues in exporting in Plot2LaTeX. Maybe you can try my version of the figure on your pc. In the zip is a m-file that creates the figure and then uses the original export function of matlab and the version with Plot2LaTeX. After that you can compile the tex-file and compare the results:

![image](https://github.com/Don-Chris/Plot2LaTeX/assets/38617646/155c3c56-da88-454c-a3c6-d8196e17e6c4)
[test.zip](https://github.com/user-attachments/files/16119797/test.zip)

**Comment by DSpale** [comment]
@Don-Chris The original figure which I used is the Test.fig file in the attached zip in my original message. I didn't call Plot2LaTeX with any additional options, only the savepath and filename.

However, by some more testing and your example, I was able to find out that the issue may have been with the font I set as standard font in Matlab (CMU Serif, the same font that LaTeX uses by default). With CMU Serif, both my figure and your example failed to position the axes correctly. But after switching to Times New Roman, the results looked much better, as seen in the examples in the attached zip file.

Is this behavior with non-standard fonts already a known limitation in plot2LaTeX?
[EM_wave_vis_TNR.zip](https://github.com/user-attachments/files/16120757/EM_wave_vis_TNR.zip)

**Comment by Don-Chris** [comment]
@DSpale This is an issue in the svg-export function of matlab. The SVG file created doesn't have any text after exporting, only line-segments representing text are remaining when changing the font. This is an issue for Plot2LaTeX, since it is searching for svg-text elements to update them.

I "fixed" your issue in the latest commit by checking the fontName used and changing it back to Helvetica if needed. This should be no problem, since LaTeX will update the text in the SVG based on the font selected in your LaTeX document.

What do you think, does this suffice or are there use-cases where the svg-fontname is important? I could add the fontName-info back to the svg file after exporting if needed.

**Comment by DSpale** [comment]
I just checked again, and sadly, it still does not seem to be working properly for me. However, for some reason only one of the axis gets misaligned - I attached the result I obtained by following your recreation of the example. Curiously, the only setup I found to consistenly produce results where neither the axes are misaligned nor the SVG labels do not get updated is with the previos version (2.4.1) and Times New Roman as selected font.

Concerning your question, I think a good compromise would be to offer the option to either keep the fontName-info or discard it, with it being discarded being set as the default. 
![grafik](https://github.com/Don-Chris/Plot2LaTeX/assets/74185752/7ac0912f-d50c-44e3-8180-f801f2457530)
